### PR TITLE
CLM-15583 - Move nx-list selected styling from IQ to RSC

### PR DESCRIPTION
**Pull request by regoarrarr**
https://issues.sonatype.org/browse/CLM-15583

### CODE REVIEW

**Comment by glenhunter** [comment]
Do we want to show an example?

**Comment by glenhunter** [comment]
We should add `.selected` to the table in the description.

**Comment by regoarrarr** [comment]
> Do we want to show an example?

Done in 1789d9b27186c44c3a46880a3136166e0de75ef5

**Comment by regoarrarr** [comment]
> We should add `.selected` to the table in the description.

Done in 1789d9b27186c44c3a46880a3136166e0de75ef5

File: gallery/src/styles/NxList/NxListPage.tsx [hunk 1]
```diff
@@ -75,6 +75,13 @@ const NxListPage = () =>
             <td className="nx-cell">Modifier of <code className="nx-code">.nx-list__item</code></td>
             <td className="nx-cell">Use this when you want to have a button on the far right.</td>
           </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.selected</code></td>
+            <td className="nx-cell">Utility class that goes along
+              with <code className="nx-code">.nx-list__item</code></td>
```
Comment (raveneyex) [inline gallery/src/styles/NxList/NxListPage.tsx:81]: I'm getting lint errors on this line — `Closing tag of a multiline JSX expression must be on its own line`

File: gallery/src/styles/NxList/NxListPage.tsx [hunk 2]
```diff
@@ -75,6 +75,13 @@ const NxListPage = () =>
...
+            <td className="nx-cell">Use the <code className="nx-code">selected</code> class
+              alongside <code className="nx-code">.nx-list__item</code> when a clickable list item is selected</td>
```
Comment (raveneyex) [inline gallery/src/styles/NxList/NxListPage.tsx:83]: Getting lint errors in this line too — `Closing tag of a multiline JSX expression must be on its own line`

**Comment by raveneyex** [comment]
I don't think it was introduced in this PR but I guess this is the place to raise it: 
In the clickable list example, the list elements have varying heights. Is that ok?

clickableListExample

**Comment by glenhunter** [comment]
> I don't know if it was introduced in this PR but I guess this is the place to raise awareness about it:
> In the clickable list example, the list elements have varying heights. Is that ok?
> 
> clickableListExample

It was purposeful when we created the example, to demonstrate how the chevron is displayed in list rows of various heights.

File: gallery/src/styles/NxList/NxListPage.tsx (hunk 2 above)
Comment (regoarrarr) [reply gallery/src/styles/NxList/NxListPage.tsx]: > Getting lint errors in this line too — `Closing tag of a multiline JSX expression must be on its own line`

Fixed in 5c769c5f7e9a5d583ad85a3dc0ec2a679036466a

File: gallery/src/styles/NxList/NxListPage.tsx (hunk 1 above)
Comment (regoarrarr) [reply gallery/src/styles/NxList/NxListPage.tsx]: > I'm getting lint errors on this line — `Closing tag of a multiline JSX expression must be on its own line`

Fixed in 5c769c5f7e9a5d583ad85a3dc0ec2a679036466a

**Comment by regoarrarr** [comment]
> Requires version bump in order to run the CI jobs

Yeah - I usually do that towards the end in case someone else bumps while I'm in the middle of the PR

05aeac63852fec69c93387e549ebda8c2e74f2c8

**Pull request merged**